05-rx-js/src/services/http.service.js

```diff
@@ -0,0 +1,20 @@
+import { fromPromise } from "rxjs/internal-compatibility";
+import { map } from "rxjs/operators";
+import { Post } from "../models/post.model";
+
+export class HttpService {
+    static getData(url) {
```

Comment:
You don't really need a class if you're never instatiating it (and you don't).
By using a static method here and importing it like this `import { HttpService as httpService }` you're basically covering the fact that you could just have a plain object exported here, or even a single function, because you only have one method.
No need to make thing more complicated than they are.